packages/client/components/TeamPrompt/TeamPromptOptions.tsx

```diff
@@ -49,22 +55,31 @@ const TeamPromptOptions = (props: Props) => {
     meetingRef
   )
 
+  const popTooltip = () => {
+    openCopiedTooltip()
+    setTimeout(() => {
+      closeCopiedTooltip()
+    }, 2000)
```

Comment:
+1 we should have a const somewhere to avoid magic numbers across the app

Reply:
That's fair! Pulled this out into a const